common/src/main/java/com/wynntils/features/trademarket/TradeMarketPriceMatchFeature.java

```diff
@@ -44,6 +46,10 @@ public class TradeMarketPriceMatchFeature extends Feature {
     private static final int PRICE_SET_ITEM_SLOT = 12;
     private static final int PRICE_INFO_ITEM_SLOT = 17;
 
+    @Persisted
+    public final Config<Boolean> silverbullTax = new Config<>(false);
```

Comment:
This syncing is not necessary, and problematic. You should have only one of these; remember, the config should really have been in the model, and will be moved there as soon as we can.  When that happen, we will have a hard time handling a "split" config like this.

Reply:
What problems does this raise? We can think of this split config as just one config with two places that allow you access it. Since they will always be the same no matter what (synced on launch, on change, etc).

If we only have one, we will have to tell people to check one config or the other when they ask about how to set silverbull tax for -t and for the price match button.